index.html

```diff
@@ -460,31 +460,30 @@ <h2>
         </p>
       </div>
       <p>
-        A <a>URL</a> <var>target</var> is said to be <dfn>within scope</dfn> of
-        <a>navigation scope</a> <var>scope</var> if the following algorithm
-        returns <code>true</code>:
+        A [=URL=] |target:URL| is said to be <dfn data-dfn-for="URL"
+        data-export="">within scope</dfn> of [=URL=] |scope:URL| if the
+        following algorithm returns `true`:
       </p>
-      <ol>
-        <li>Let <var>scopePath</var> be the [=string/concatenation=] of
-        <var>scopes</var>'s <a data-cite="URL#concept-url-path">path</a>, using
-        U+002F (/).
+      <ol class="algorithm">
+        <li>If |target| and |scope| are not [=same origin=], return `false`.
+        </li>
+        <li>Let |scopePath:string| be the [=string/concatenation=] of
+        |scopes|'s [=URL/path=] and U+002F (/).
         </li>
-        <li>Let <var>targetPath</var> be the [=string/concatenation=] of <var>
-          target</var>'s <a data-cite="URL#concept-url-path">path</a>, using
-          U+002F (/).
+        <li>Let |targetPath:string| be the [=string/concatenation=] of
+        |target|'s [=URL/path=] and U+002F (/).
         </li>
-        <li>If <var>target</var> is <a>same origin</a> as <var>scope</var> and
-        <var>targetPath</var> starts with <var>scopePath</var>, return
-        <code>true</code>.
+        <li>If |targetPath| starts with |scopePath:string|, return `true`.
         </li>
-        <li>Otherwise, return <code>false</code>.
+        <li>Otherwise, return `false`.
         </li>
       </ol>
       <p>
-        A <a>URL</a> <var>target</var> is said to be <dfn data-lt=
-        "within-scope-manifest">within scope of a manifest</dfn>
-        <var>manifest</var> if <var>target</var> is <a>within scope</a> of the
-        navigation scope of <var>manifest</var>.
+        A [=URL=] |target:URL| is said to be <dfn data-export="" data-dfn-for=
+        "manifest" data-lt="within scope">within the navigation scope of a web
```

Comment:
Can this term just be "within scope", otherwise it is again too verbose (as it is intended as a way to write a short-hand).

"**within the navigation scope of a web manifest** _manifest_" is longer than "**within scope** of the navigation scope of _manifest_", so there is no point using this short-hand if its name is that long.

Reply:
Well, we don't ever need to write "within the navigation scope of a web manifest"... it's only defined as that here. In every other spec or local instance, one just writes [=manifest/within scope=].

Reply:
(Not bothered with what we have to write in the source code, but what it reads like.)

Does `[=manifest/within scope]=` render as "within scope" or "within the navigation scope of a web manifest"? If the former, I am happy.

However, I would still like to define this as "within scope of a web manifest" rather than "within the navigation scope of a web manifest" (the latter is too verbose and doesn't match the previous definition, "within scope of a URL").

Reply:
I took all the above (plus the a good weekend break) and rewrote various things...